packages/atomic/src/pages/test-no-search.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
Wy are only the ResultList tests using this file?

Reply:
Because it's the first to test the its "placeholder", but there will be more components with placeholder to test